install_config/aggregate_logging.adoc

```diff
@@ -96,10 +96,16 @@ be applied to their deployment configurations.
 [[aggregate-logging-ansible-variables]]
 == Specifying Logging Ansible Variables
 
-Parameters for the EFK deployment may be specified to the
- xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory host file]
-to override the
- https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/defaults/main.yml[default parameter values].
+You can override the https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/defaults/main.yml[default parameter values]
+by specifying parameters for the EFK deployment in the
+ xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory host file].
+
+[NOTE]
+====
+Some variables require you to specify a value in certain situations, such as if there is no default value in the Ansible role.
+See the descriptions of `openshift_logging_es_pvc_size` and `openshift_logging_es_pvc_dynamic` for details.
+====
```

Comment:
@jhoyt-rh This note would scare me a bit where we don't list all of the variables that require a value and in which situations. Can we omit the note?

Reply:
Based on your feedback, I removed the note and put the info in the parameter descriptions. I will  need to discuss with the CSS folks, since they asked about adding this.